static/app/components/group/assignedTo.tsx

```diff
@@ -158,7 +162,7 @@ export function getAssignedToDisplayName(group: Group) {
     return user?.name ?? group.assignedTo.name;
   }
 
-  return group.assignedTo?.name ?? t('No one');
+  return group.assignedTo?.name ?? isFeedback ? t('Unassigned') : t('No one');
```

Comment:
Lets move the `t('No one')` and `t('Unassigned')` translations into the callsites, then we don't need to pass in `isFeedback` at all.

So the code would look like this in all the places:
```
import {getAssignedToDisplayName} from '...';

return <span>{getAssignedToDisplayName(issue) ?? t('No one')}</span>;
```

this changes the function return type from `string` to `string | undefined` but de-couples the function from that string which is what we want.